ProjectSettings/ProjectSettings.asset

```diff
@@ -118,7 +118,7 @@ PlayerSettings:
     16:10: 1
     16:9: 1
     Others: 1
-  bundleVersion: 2.5.0
+  bundleVersion: 2.6.0
```

Comment:
Looks like this is bringing in some mrtk_dev changes :/

Reply:
Yep mrtk_dev was merged with https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8386/commits/0b02d55f9f1794775dcb33a742ab26044c5a3171

Reply:
will clean

Reply:
You're probably gonna need to rebase out the merge commit from this branch. Committing https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8386/commits/177f4df6ce0a28cdc1d56b9d0976e05e87583b46 will revert the changes in mrtk_dev when we sync stabilization to dev, which we don't want.

Reply:
finished the big rebase 💦